GradeCenter.API/Controllers/SchoolController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using GradeCenter.API.Models.Request;
+using GradeCenter.Services.Schools;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GradeCenter.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SchoolController : ControllerBase
+    {
+        private readonly ISchoolService _schoolService;
+
+        public SchoolController(ISchoolService schoolService)
+        {
+            _schoolService = schoolService;
+        }
+
+        /// <summary>
+        /// Creates an object of type School and stores it in the database.
+        /// </summary>
+        /// <param name="requestModel"></param>
+        /// <returns></returns>
+        [HttpPost("Create")]
+        public async Task<IActionResult> Create(SchoolCreateRequestModel requestModel)
+        {
```

Comment:
Here you will also need to validate the request model values and that if the user is authenticated and throw "BadRequest()"  or "Unauthorized()" as response, same for other EndPoints.